libraries/rush-lib/src/logic/pnpm/PnpmOptionsConfiguration.ts

```diff
@@ -432,9 +434,16 @@ export class PnpmOptionsConfiguration extends PackageManagerOptionsConfiguration
     jsonFilename: string,
     commonTempFolder: string
   ): PnpmOptionsConfiguration {
-    const pnpmOptionJson: IPnpmOptionsJson = JsonFile.loadAndValidate(
-      jsonFilename,
-      PnpmOptionsConfiguration._jsonSchema
+    // TODO: plumb through the terminal
```

Comment:
Hmm, I'm guessing this isn't a super easy task?

Reply:
Yeah it isn't.